src/verinfast/dependencies/walkers/classes.py

```diff
@@ -65,12 +65,18 @@ def __init__(
         if print_name:
             logger(print_name)
         self.files = []
-        self.manifest_files = manifest_files.copy()
-        for f in self.manifest_files:
+        new_files = []
+        for f in manifest_files:
             if "*" in f:
-                expanded = glob.glob(f, root_dir=root_dir)
-                self.manifest_files.remove(f)
-                self.manifest_files += expanded
+                # Expand the wildcard to get full paths
+                full_paths = glob.glob(os.path.join(root_dir, f))
```

Comment:
This isn't necessary, glob has a root_dir argument

Reply:
With Python 3.9 it throws this error:
TypeError: glob() got an unexpected keyword argument 'root_dir'